addons/attributes/initAttributes.sqf

```diff
@@ -518,7 +526,8 @@
             _x setWaypointType _type;
             if (_type == "SCRIPTED") then {_x setWaypointScript _script};
         } forEach SELECTED_WAYPOINTS;
-    }
+    },
+    {GVAR(enableWpName)}
```

Comment:
Missing setting for this? Also, should be `enableWaypointType`.

Reply:
ah oops, yeah I was gonna stop at obj and grp

Reply:
Why not include waypoints and markers as well, for completeness.